app/assets/stylesheets/views/_completed-transaction.scss

```diff
@@ -39,7 +39,7 @@
 
     textarea {
       display:block;
-      @include core-19;
+      @include govuk-font(19);
       margin: 0 0 1em 0;
       background-color: #f8f8f8;
       border: 1px inset #bfc1c3;
```

Comment:
can it have either `$govuk-border-colour | #b1b4b6 ` or `$govuk-input-border-colour | #0b0c0c` depending on how other inputs look like?